router/http/src/main/scala/io/buoyant/router/Http.scala

```diff
@@ -1,21 +1,20 @@
 package io.buoyant.router
 
-import com.twitter.finagle.{Http => FinagleHttp, Server => FinagleServer, http => _, _}
 import com.twitter.finagle.client.StackClient
 import com.twitter.finagle.http.{Request, Response, TlsFilter}
 import com.twitter.finagle.param.ProtocolLibrary
 import com.twitter.finagle.server.StackServer
-import io.buoyant.router.http.{ForwardedFilter, Identifier}
+import com.twitter.finagle.{Http => FinagleHttp, Server => FinagleServer, http => fhttp, _}
+import io.buoyant.router.Http.param.HttpIdentifier
+import io.buoyant.router.http.{ForwardedFilter, DefaultIdentifier$}
```

Comment:
Pretty sure this import should be:

    import io.buoyant.router.http.{ForwardedFilter, DefaultIdentifier}